### PR TITLE
Revert "[WIP] Travis CI + PSR4 Autoloading"

### PR DESCRIPTION
**Pull request by isublimity**
Reverts smi2/phpClickHouse#49

### CODE REVIEW

**Comment by simPod** [comment]
Hi @isublimity 

I saw some commits undoing my changes from https://github.com/smi2/phpClickHouse/pull/49. Would you like to explain some of them? I'm willing to consult it with you so we don't undo something that should not be.

For example, I noticed this commit https://github.com/smi2/phpClickHouse/commit/b0a00e5fac614b0ee40c9040ddae3438a13ba0b8

Those files you added back are not project related but your local machine related. You should add them to gitignore global instead https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

**Comment by simPod** [comment]
How do you use include.php? I guess it could be in the project as a standalone file while using autoloading via composer 🤔 But TBH haven't seen doing this since started working with PHP. Is that for legacy reasons in your main project?

Using different PHP version for dev is not possible AFAIK. Maybe `--ignore-platform-requirements` flag might help you.

If those changes cause problems for you, we could eg. start `develop` branch where I would be merging my updates. It can then be merged to master later.

**Comment by isublimity** [comment]
"php": "^7.1"
    "phpunit/phpunit": "^7.0"

**Comment by simPod** [comment]
I see the pull request was reverted on master (maybe hard reset?). But in case you have the code I sent to merge somehow checked out:
 
1) check if the file is present https://github.com/simPod/phpClickHouse/blob/travis-ci/tests/WithClient.php, 
2) then don't forget to run `composer install`. It should setup PSR-4 autoloading. 
3) Then you can simply run `vendor/bin/phpunit` and it should output the following

> PHPUnit 7.1.4 by Sebastian Bergmann and contributors.
>
> Runtime:       PHP 7.2.5 with Xdebug 2.6.0
> Configuration: /path/phpClickHouse/phpunit.xml.dist

I'd not recommend running phar if we have it all setup using composer.

**Comment by isublimity** [comment]
I check my project codes, and merge to master 

Change list:
```
Move \ClickHouseDB\WhereInFile to \ClickHouseDB\Query\WhereInFile
Move \ClickHouseDB\QueryException to \ClickHouseDB\Exception\QueryException
Move \ClickHouseDB\DatabaseException to ClickHouseDB\Exception\DatabaseException
Move \ClickHouseDB\FormatLine to \ClickHouseDB\Quote\FormatLine
Move \ClickHouseDB\WriteToFile to ClickHouseDB\Query\WriteToFile
Move \Curler\Request to \ClickHouseDB\Transport\CurlerRequest
Move \Curler\CurlerRolling to \ClickHouseDB\Transport\CurlerRolling
```

**Comment by simPod** [comment]
If you setup travis, it will check it for you if there's no mistake ;) But looks ok at first glance

**Comment by simPod** [comment]
https://github.com/mbonaci/mbo-storm/wiki/Integrate-Travis-CI-with-your-GitHub-repo#integrate-your-github-project-with-travis-ci

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-PHP ClickHouse wrapper
-===================
+PHP ClickHouse driver
```
Comment (simPod) [inline README.md]: `Client` I would say

File: README.md [hunk 2]
```diff
@@ -29,6 +29,9 @@ composer require smi2/phpclickhouse
 
 ### OR install submodule
 
+`composer` require php 7.1, but phpClickHouse can work on php 5.5 without autoload and `composer`
```
Comment (simPod) [inline README.md]: Just note this http://php.net/supported-versions.php Shouldn't encourage 5.5 here ;)

File: README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 PHP ClickHouse wrapper
```
Comment (simPod) [inline README.md]: I still am persuaded this is more like client and not a wrapper as it doesn't wrap anything ;)

**Comment by isublimity** [comment]
@simPod Let us apply this PR to the master, then fix that code style

**Comment by simPod** [comment]
Yup but I'd prefer to apply my comments to this one before merge, especially those redundant comments

**Pull request merged**